marklogic-data-hub-central/ui/src/components/entities/merging/merge-strategy-dialog/merge-strategy-dialog.module.scss

```diff
@@ -31,6 +31,12 @@
     margin-left: 12px;
 }
 
+.questionCircleDefault {
+  font-size: 12px;
+  color: #7f86b5;
```

Comment:
Can we use an variable in HCVariables.scss file? With the HCUtils.scss file imported, you should be able to use all the variables present in HCVariables.scss

Reply:
Good catch @yubiselmarklogic . Made the fix, thanks.